packaging/windows/OpenRA.nsi

```diff
@@ -227,7 +229,19 @@ FunctionEnd
 !insertmacro Clean "un."
 
 Section "Uninstall"
+	${nsProcess::FindProcess} "OpenRa.Game.exe" $R0
+	IntCmp $R0 0 gameRunning
+	${nsProcess::FindProcess} "OpenRa.exe" $R0
+	IntCmp $R0 0 gameRunning
+	${nsProcess::FindProcess} "OpenRa.Editor.exe" $R0
+	IntCmp $R0 0 gameRunning
+	${nsProcess::Unload}
 	Call un.Clean
+	Goto end
+	gameRunning:
+		MessageBox MB_OK|MB_ICONEXCLAMATION "OpenRa is running. Please close it first" /SD IDOK

```

Comment:
I don't know if Windows users are really that helpless and need such an advice. It's not like this is a small application that runs silently in the background.